compiler/qsc_frontend/src/lower.rs

```diff
@@ -736,6 +736,10 @@ impl With<'_> {
             new_id
         })
     }
+
+    fn lower_vec_ident(&mut self, name: &ast::VecIdent) -> hir::VecIdent {
+        name.iter().cloned().map(|i| self.lower_ident(&i)).collect()
```

Comment:
Is the cloned needed here also? Or does `name.iter().map(|i| self.lower_ident(i)).collect()` work?